.github/CODEOWNERS

```diff
@@ -0,0 +1,36 @@
+# See github docs for more info on the syntax:
```

Comment:
I'd probably not use CODOWNERS for ovirt-engine-4.4 branch, there are usually much stricter permissions who can merge stuff into stable branch